src/style.css

```diff
@@ -1,3 +1,79 @@
 body {
   margin: 0;
+  max-width: 1200px;
```

Comment:
It would be great not to limit the max width, so the header looks okay on large screens too
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/15f973f9-154e-484f-bc28-8d62b04c42ee)